packages/web-pkg/src/composables/localStorage/useLocalStorage.ts

```diff
@@ -1,14 +1,14 @@
-import { ref, watch, unref } from 'vue'
+import { ref, watch, unref, Ref } from 'vue'
 
-export const useLocalStorage = (key: string, defaultValue: any = undefined): any => {
+export const useLocalStorage = <T>(key: string, defaultValue: T = undefined) => {
   const existingValue = window.localStorage.getItem(key)
   const variable = ref(defaultValue)
 
   if (existingValue) {
     try {
       variable.value = JSON.parse(existingValue)
     } catch {
-      variable.value = existingValue
+      ;(variable as Ref<string>).value = existingValue
```

Comment:
This doesn't look like it should be necessary or even completely correct

Reply:
Problem is that `existingValue` is a string, so in that catch-phrase here `variable` will eventually be a string as well.